shared_model/backend/protobuf/from_old_model.hpp

```diff
@@ -53,6 +53,7 @@ namespace shared_model {
         const iroha::model::Proposal &proposal) {
       iroha::protocol::Proposal proto;
       proto.set_height(proposal.height);
+      proto.set_created_time(proposal.created_time);
```

Comment:
Why don't we use builders?

Reply:
Not all of the fields was set, I think it can be changed now